README.md

```diff
@@ -183,4 +183,26 @@ Congratulations, your project is ready to go!
 
 ## Troubleshooting
 
-TODO
+### Debug logging
+
+To troubleshoot MongoDB connectivity issues you can enable PyMongo's logging feature with Django's `LOGGING` setting.
```

Comment:
- Please wrap lines at 80 characters.
- missing comma after "issues"
- chop "feature"
- It would be nice to include links to https://pymongo.readthedocs.io/en/stable/examples/logging.html and https://docs.djangoproject.com/en/stable/topics/logging/ as in https://github.com/Snowflake-Labs/django-snowflake/blob/1b867efa8924dc50c63149f099b8d084eaabc13e/README.md#L123-L125